src/protos/grpc/BUILD.bazel

```diff
@@ -26,4 +26,48 @@ cc_grpc_library(
     srcs = [":master_metadata_service_proto"],
     deps = [":cc_master_metadata_service_proto"],
     grpc_only = True,
-)
\ No newline at end of file
+)
+
+proto_library(
+  name = "chunk_server_lease_service_proto",
+  visibility = ["//visibility:public"],
+  srcs = ["chunk_server_lease_service.proto"],
+  deps = [
+    "@com_google_protobuf//:timestamp_proto",
+  ]
+)
+
+cc_proto_library(
+  name="cc_chunk_server_lease_service_proto",
+  visibility = ["//visibility:public"],
+  deps=[":chunk_server_lease_service_proto"],
+)
+
+cc_grpc_library(
+    name = "cc_chunk_server_lease_service_grpc",
+    visibility = ["//visibility:public"],
+    srcs = [":chunk_server_lease_service_proto"],
+    deps = [":cc_chunk_server_lease_service_proto"],
+    grpc_only = True,
+)
+
+
+proto_library(
+  name = "chunk_server_file_service_proto",
+  visibility = ["//visibility:public"],
+  srcs = ["chunk_server_file_service.proto"],
+)
+
+cc_proto_library(
+  name="cc_chunk_server_file_service_proto",
+  visibility = ["//visibility:public"],
+  deps=[":chunk_server_file_service_proto"],
+)
+
+cc_grpc_library(
+    name = "cc_chunk_server_file_service_grpc",
```

Comment:
nit: indentation not uniform with others

Reply:
good catch. it turns out my formatting extension has some permission issues, so now i fixed it because of you pointing it out